### ISSUE TITLE
Followers Stat is Broken on Client

### ISSUE THREAD

**Issue by Dragonify** [gamejolt/next-issue-tracker#1229]
Before I describe the issue, I just want to say that what you're doing is absolutely amazing. Thank you for maintaining Game Jolt so well and keep users like us satisfied and happy. I love the latest update, I just need to get used to it. I feel like I'm reading the website upside-down when the comment on a game page is now at the left side haha. Anyway, back to the point,

My profile stat is not working correctly. In fact, every user's profile stat I'm seeing is incorrect. It just says I (and other people) have 1 following and 1 follower. This only happens in Game Jolt Client.

Here's some screenshots in case you don't get what I mean.
![game jolt](https://user-images.githubusercontent.com/34051788/49323824-cd99b880-f55c-11e8-8490-b533f6052f66.jpg)
![gj](https://user-images.githubusercontent.com/34051788/49323827-d4c0c680-f55c-11e8-858e-ca020f722559.jpg)

**Comment by nilllzz** [comment]
Hm, I can't confirm this.
Which version of the client are you running?
You can find the version number in the footer.
![image](https://user-images.githubusercontent.com/2119926/49331684-62260a00-f5a1-11e8-8657-95f5d54c6df4.png)
If it's anything below 0.26.0, you should try to update it :)

Edit: also, thanks for the kind words, glad you are enjoying our updates.

**Comment by Dragonify** [comment]
Yes, this is the latest version. And I should add that this happened right after I updated the Client.
![screenshot 133](https://user-images.githubusercontent.com/34051788/49335040-f931a700-f61f-11e8-8b57-b5457447611d.png)

**Comment by YLivay** [comment]
I think you simply toggled the translations by accident.

scroll down to the footer and see what its set as.

![image](https://user-images.githubusercontent.com/996947/49397854-70ad2680-f745-11e8-9a4f-d065fb6c5038.png)

**Comment by Dragonify** [comment]
Huh, it's fixed now after I changed it back to English. But why does it effect the problem? No, I didn't toggle 
it by accident. In fact, I've been using my language (Chinese) for a while. And before the update, there was no problem.

**Comment by YLivay** [comment]
We've been having problems extracting the strings that need translation, so new terms are just left in the language they were originally written in, sorry about that :(

I'm not sure we have an ETA on it, we'll raise this in the next sprint to see if we can prioritize this.
